### DISCUSSION TITLE
cancel-progress-programmatically-visual-studio-code-extensions/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#609] [Comments]
# cancel-progress-programmatically-visual-studio-code-extensions/

Azure / Microsoft 365 / Visual Studio Code / Speaker / Blogger

https://www.eliostruyf.com/cancel-progress-programmatically-visual-studio-code-extensions/

**Comment by AxelGauffre** [comment]
Hi Elio,
Thank you for all you articles.
They often help me a lot.

I asked this question on SO and I have the feeling you could help me there: https://stackoverflow.com/questions/75278473/how-to-prevent-the-foldingrangeprovider-from-executing-after-i-modify-some-text/75278535
